components/org.wso2.carbon.datasource.core/src/main/java/org/wso2/carbon/datasource/rdbms/hikari/utils/HikariDataSourceUtils.java

```diff
@@ -47,6 +48,12 @@ public static HikariConfig buildConfiguration(String xmlConfiguration) throws Da
             config.setMaximumPoolSize(configuration.getMaximumPoolSize());
             config.setMinimumIdle(configuration.getMinimumIdle());
             config.setAutoCommit(configuration.isAutoCommit());
+            if (configuration.getDatabaseProps() != null && !configuration.getDatabaseProps().isEmpty()) {
+                Properties properties = new Properties();
+                configuration.getDatabaseProps()
+                        .forEach(property -> properties.setProperty(property.getName(), property.getValue()));
```

Comment:
Line break should have to be before the dot. That is the java 8 standard we have followed. Example[1].

[1] https://github.com/wso2/carbon-kernel/blob/master/core/src/main/java/org/wso2/carbon/kernel/internal/startupresolver/StartupOrderResolver.java